src/rules/preferForOfRule.ts

```diff
@@ -17,7 +17,7 @@
 
 import * as ts from "typescript";
 import * as Lint from "../index";
-import { isAssignment } from "../language/utils";
+import { isAssignment, unwrapParentheses } from "../language/utils";
```

Comment:
don't know if this is a requirement, but all other rules use the functions from the `Lint` namespace. for consistency you could use `Lint.isAssignment` and `Lint.unwrapParentheses` here